Lib/glyphsLib/builder/groups.py

```diff
@@ -140,12 +140,28 @@ def _to_glyphs_kerning_group(self, name, glyphs):
 
 def _glyph_kerning_attr(glyph, side):
     """Return leftKerningGroup or rightKerningGroup depending on the UFO
-    group's side (1 or 2).
+    group's side (1 or 2) and writing script.
     """
-    if int(side) == 1:
-        return "rightKerningGroup"
-    else:
-        return "leftKerningGroup"
+    if glyph.parent.format_version > 2:  # Glyphs 3
+        glyph_data = glyphdata.get_glyph(glyph.name)
+        if (
+            glyph_data.script in ("arabic", "hebrew")
```

Comment:
why only "arabic" and "hebrew"?

Reply:
These are the two RTL scripts to my knowledge. This point is arguably the critical one, also about the numbers. I chose those to my best knowledge. Some algorithm needs to be in place to sort Glyphs’ kerning classes into the correct first/second pairs, and I didn’t know any better.

Reply:
I see, there are more RTL scripts. Are those already defined somewhere in the toolchain for fonts?

Reply:
You can get script direction from a function in ufo2ft.utils.

(I’m on the road this week so won’t be able to review.)

Reply:
> I can add the writing direction to the glyphdata.xml but there might be a way to query unicode to get the writing direction of each code. Then we don’t need to check for `script == arabic and category != Number`.

Yes, I could try this approach. @anthrotype Seems like a good idea?